api-report/mocha-test-setup.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-/// <reference types="mocha" />
```

Comment:
I keep getting this on build - not sure if someone failed to include it on a prior PR or if I'm doing something wrong.  LMK if I'm the only one and I'll drop the change from here.